packages/shared/forks/ReactFeatureFlags.www.js

```diff
@@ -15,6 +15,7 @@ import typeof * as DynamicFeatureFlags from './ReactFeatureFlags.www-dynamic';
 const dynamicFeatureFlags: DynamicFeatureFlags = require('ReactFeatureFlags');
 
 export const {
+  disableLegacyMode: disableLegacyModeDynamic,
```

Comment:
Why the name mapping?

Reply:
because we export below on line 123 where we set it statically to true for `__EXPERIMENTAL__` (so the modern build will have the true baked in like before)

Reply:
Actually, I did mess this up. Good catch